src/shared/logs-show.c

```diff
@@ -511,13 +544,18 @@ static int output_short(
                 if (r < 0)
                         return r;
         }
+
```

Comment:
Please drop this. We usually write error handling without inserting an empty line.